### ISSUE TITLE
Create WCAG compliant colour pallete

### ISSUE THREAD

**Issue by ADubhlaoich** [ADubhlaoich/swing-ie#2]
Right now the website doesn't fire good warnings as a result of its colour contrast - design isn't my strong point nor is design an important facet of this website, but it would be hypocritical of me to do accessibility research while leaving my own projects inaccessible.

**Comment by ADubhlaoich** [comment]
I'll be appropriating Swing Focus's colour palette to the website as part of the refactor, which should be colour compliant. Should is a very dangerous word, though!
